src/components/reference-btn-group.tsx

```diff
@@ -0,0 +1,94 @@
+import {useEffect, useState} from 'react'
+import css from 'styled-jsx/css'
+
+import Button from './button'
+
+export const enum ReferenceType {
+  NONE = 'NONE',
+  THE_SELECT = 'THE_SELECT',
+  THIS_PAGE = 'THIS_PAGE',
+}
+
+interface PropsType {
+  parentType?: ReferenceType
+  disabled?: boolean
+  onTypeChange?: (bnt: ReferenceType) => void
+}
+
+export default function ReferenceBtnGroup({parentType, disabled, onTypeChange}: PropsType) {
+  const [type, setType] = useState<ReferenceType>(parentType)
```

Comment:
This state is unnecessary, you can directly use 'parentType' to do some processing.

Reply:
all internal state removed, use external state control this component.